src/components/extensive/Tables/DisturbancesTable.tsx

```diff
@@ -35,12 +41,42 @@ const DisturbancesTable = ({ modelName, modelUUID, collection, onFetch }: Distur
   );
 
   const hasAllHeaders = !!disturbances?.data?.[0]?.extent && !!disturbances?.data?.[0].intensity;
+  const { data: record } = useGetV2FormsENTITYUUID<{ data: GetV2FormsENTITYUUIDResponse }>({
+    pathParams: {
+      uuid: modelUUID,
+      entity: modelName
+    }
+  });
+
+  const viewDataDisturbances = record?.data?.form?.form_sections.map((questions: any) =>
+    questions.form_questions.map((item: any) => item.uuid).map((item: any) => record?.data?.answers?.[item])
```

Comment:
I think is better to have it named as is `formQuestion` in the first `map` and later use the formQuestionUUID in the second `map`